### PR TITLE
Get squads backend connected

### PR DESCRIPTION
**Pull request by vivekjain1397**
-Retrieves squads from backend 
-Passes squad id, squad name, and squad emoji to events page.

### CODE REVIEW

File: mobile/backend/backend.tsx [hunk 1]
```diff
@@ -6,14 +6,11 @@ export const BACKEND_URL = "http://localhost:5000/"
 export const callBackend = async (endpoint: string, init: RequestInit = { headers: {} }) => {
     const sessionCookie = await AsyncStorage.getItem("sessionCookie")
     init.headers["Cookie"] = sessionCookie
-    init.credentials = "include"
+    // init.credentials = "include"
```
Comment (abhishyantkhare) [inline mobile/backend/backend.tsx:9]: I think we still need this

File: mobile/squads/squads.tsx [hunk 2]
```diff
@@ -1,67 +1,33 @@
...
   name: string,
-  emoji: string,
-  email: string,
-  events: Event[]
+  squad_emoji: string,
 }
 
 const dummySquads: Squad[] = [
   {
+    id: 1,
```
Comment (abhishyantkhare) [inline mobile/squads/squads.tsx:18]: Do we need this anymore? I think we can get rid of dummySquads

**Comment by abhishyantkhare** [comment]
Will approve once u add back in credentials=include

File: mobile/backend/backend.tsx (hunk 1 above)
Comment (vivekjain1397) [reply mobile/backend/backend.tsx]: done.

File: mobile/squads/squads.tsx [hunk 3]
```diff
@@ -85,25 +31,39 @@ const Squads = (props) => {
...
   }
 
-  const goToEvents = (events: Event[], squadName: string, squadEmoji: string, email: string) => {
+  const goToEvents = (id: number, name: string, squad_emoji: string) => {
     props.navigation.navigate("Events", {
-      events: events,
-      squadName: squadName,
-      squadEmoji: squadEmoji,
-      email: email
+      squadId: id,
```
Comment (c0dyzeng) [inline mobile/squads/squads.tsx:56]: thank you 😍

**Pull request merged**